kong/plugins/jwt/daos.lua

```diff
@@ -21,6 +21,12 @@ local SCHEMA = {
     if plugin_t.algorithm == "RS256" and crypto.pkey.from_pem(plugin_t.rsa_public_key) == nil then
       return false, Errors.schema "'rsa_public_key' format is invalid"
     end
+    if plugin_t.algorithm == "RS512" and plugin_t.rsa_public_key == nil then
+       return false, Errors.schema "no mandatory 'rsa_public_key'"
```

Comment:
Good catch! @sarraz1 Any updates here? Those style issues are all that's left for us to merge it, if you address them we won't hesitate to push the button :)